### ISSUE TITLE
Scaffold a new project using the CLI

### ISSUE THREAD

**Issue by dOrgJelli** [polywrap/documentation#114]

**Comment by dOrgJelli** [comment]
Relevant documentation:
https://docs.polywrap.io/guides/create-as-wrapper/project-setup#create-your-project
https://docs.polywrap.io/devtools/polywrap-cli#create-c

**Comment by dOrgJelli** [comment]
Requesting review: @evanjacobs @kev-ngo

**Comment by evanjacobs** [comment]
The documentation at [https://docs.polywrap.io/devtools/polywrap-cli](https://docs.polywrap.io/devtools/polywrap-cli) looks like it could be updated.

Here are some issues that I noticed:
1. Inconsistent headings and formatting. In some cases, we use the heading "Example" and in other cases it is "Commands" although the usage of each is very similar. We should standardize on one approach.
2. There aren't examples of "create app" or "create plugin"

Related: I noticed an issue on another docs page. On [https://docs.polywrap.io/guides/create-js-dapp/install-client](https://docs.polywrap.io/guides/create-js-dapp/install-client), I believe the language argument should be `typescript-react` and not `react`.
